style.css

```diff
@@ -321,6 +336,7 @@ border-radius: 8px;
   width: 20.44rem;
   height: 6.3rem;
 }
+
```

Comment:
- [ ] Kindly observe the huge difference between your about section and the figma's? I suggest you try to check the figma design again for accurate units of spacing, height, width, etc so that it can match design as shown below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219014907-2d2ce505-d4f0-4348-97db-f858a36ff6dd.png)|![image](https://user-images.githubusercontent.com/99715640/219015036-4049eff0-9fd5-48f9-9ba6-12d7fc699832.png)|

Reply:
removed the heights.